lib/soapy_cake/admin_batched.rb

```diff
@@ -28,7 +28,9 @@ def initialize(admin, method, opts, limit)
       def to_enum
         Enumerator.new do |y|
           loop do
-            result = admin.public_send(method, opts.merge(row_limit: limit, start_at_row: offset))
+            result = admin
+              .public_send(method, opts.merge(row_limit: limit, start_at_row: offset))
+              .to_enum
```

Comment:
Actually it hurts a lot, because if it's not an enumerator already, all the nice benefits of an enumerator are lost. So let's please rather fix the tests.